lib/core/configuration.dart

```diff
@@ -103,6 +104,7 @@ class Configuration {
     }
     await _preferences.clear();
     await _secureStorage.deleteAll();
+    await UserService.instance.logout();
```

Comment:
Let's await on this and verify the result before clearing `_preferences` or `_secureStorage`? Just to be safe?